basic/stam/stam_exercise.py

```diff
@@ -0,0 +1,41 @@
+import sys
+import os
+
+PATH = 1
```

Comment:
Don't need that.
Don't use consts on 1 use only

Reply:
I ommited the PATH, I beieve the import is ok